roles/common/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Verify pod - {{ common_pod_test_id }}"
```

Comment:
This test id should be elsewhere. need to check the compute logging playbook

Reply:
The reason why the test_id shouldn't be here is because the include_tasks task will pass as long as the task file exists.